### DISCUSSION TITLE
no type hint checking

### DISCUSSION THREAD

**Discussion by stevenlis** [microsoft/pylance-release#3619] [Q&A] [answered]
I'm getting no type hint checking, I tried but unable to find any doc for this
I'm on Python 3.9.12
![CS-h3SDf5oz](https://user-images.githubusercontent.com/33796896/201504646-8c97cf6c-8786-4ce5-b710-bdb05fd550bd.png)

I already turn the following feature on
``` "python.analysis.typeCheckingMode": "basic"```

Am I missing something?

**Comment by rchiodo** [comment, marked as answer]
This is working for me:

![image](https://user-images.githubusercontent.com/19672699/201725355-e155f1f4-ff4a-47d5-8f2a-9b56bab8ed75.png)


Can you paste your user settings.json? Perhaps pylance is turned off?

**Comment by stevenlis** [comment]
@rchiodo I forgot that I have a workspace settings.json that is different from my global ones...🤪
